packages/flutter/lib/src/material/switch.dart

```diff
@@ -441,11 +498,11 @@ class Switch extends StatelessWidget {
         height: size.height,
         alignment: Alignment.center,
         child: CupertinoSwitch(
```

Comment:
Similar to my comment about Switch's icon and color properties: in the future, perhaps we can avoid delegating to a Cupertino library widget, and just configure the Material Switch widget instead.